### ISSUE TITLE
Oracle JDK builds include proprietary javadoc

### ISSUE THREAD

**Issue by hyandell** [qos-ch/slf4j#419]
Hey Ceki and qos folk,

Looking at the Javadoc jar from https://central.sonatype.com/artifact/org.slf4j/slf4j-ext/2.1.0-alpha1 it contains:

    slf4j-ext-2.1.0-alpha1-javadoc/legal/LICENSE

This is an Oracle "No-Fee Terms and Conditions license agreement". 

I'm not sure if this was your intent, or if this has snuck in due to Oracle JDK being used to perform the release. Sharing in case it's a concern for you.

**Comment by ceki** [comment]
@hyandell Thank you for catching this.

**Comment by ceki** [comment]
How did it get there and how do I get rid of it in future releases?

**Comment by hyandell** [comment]
My theory has been that it comes from releasing with the Oracle JDK rather than OpenJDK. OpenJDK inserts GPL JavaScript for this search feature, which is less of a "Whaaatttt???" flag than the proprietary Oracle text. At least for my over-Apache influenced eyes :)

Though I saw you commented on the Fury issue that you get the proprietary licensing even when using OpenJDK?

**Comment by ceki** [comment]
@hyandell 

> Though I saw you commented on the Fury issue that you get the proprietary licensing even when using OpenJDK?

Yes. I think so. In any case, the contents of legal/LICENSE generated by OpenJDK javadocs also seemed strange but I have not read the contents of the license with any attention.

Eclipse Temurin also generates a legal/LICENSE file which contains the string "../base.legal" which may be more acceptable.

**Comment by hyandell** [comment]
My recollection is that OpenJDK's 'strangeness' in the javadoc will be GPL-2.0, I believe with ClasspathException. And Temurin is presumably pointing to an Eclipse license (BSD? Eclipse-2.0? I haven't looked).

It's very frustrating that N different approaches are all sneaking licensing into the built artifacts. I feel we all reasonably expect that the Java build tools add no licensing beyond what the input provides.

I think they are largely a nuisance, I've long since adapted to not worry about the GPL showing in javadoc jars for the search functionality. For example, not something that worries me over at Apache with a licensing committee hat on. And I think most open source projects are using OpenJDK.

But the Oracle's JDK more proprietary licensing feels out of sorts with an open source project; it's weird enough noise for users that I feel it causes more confusion/doubt than simply seeing an Eclipse or GPL license.
